dali/python/CMakeLists.txt

```diff
@@ -26,6 +26,12 @@ target_link_libraries(${dali_python_lib} PRIVATE ${CUDA_LIBRARIES} ${dali_lib})
 set_target_properties(${dali_python_lib} PROPERTIES
   LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/dali/python/nvidia/dali)
 
+string(REPLACE "." "" CUDA_VERSION_LONG ${CUDA_VERSION})
+set(CU_DALI_VERSION "")
+if (${CUDA_VERSION} VERSION_GREATER "9.2")
```

Comment:
what do we do if CUDA version is lower or equal than 9.2 ?

Reply:
We stick to the default name as it is now, so the old installation way still works.